var/spack/repos/builtin/packages/py-pydicom/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPydicom(PythonPackage):
+    """Pure python package for DICOM medical file reading and writing
+
+    pydicom is a pure Python package for working with DICOM files. It lets you
+    read, modify and write DICOM data in an easy "pythonic" way."""
+
+    homepage = "https://github.com/pydicom/pydicom"
+    pypi     = "pydicom/pydicom-2.1.2.tar.gz"
+
+    version('2.1.2', sha256='65f36820c5fec24b4e7ca45b7dae93e054ed269d55f92681863d39d30459e2fd')
+
+    variant('numpy', default=False, description='Use NumPy for Pixel data')
```

Comment:
I might suggest the variant to be called `pixel` and the description be "Enable support for Pixel data".

Reply:
That was my initial thought. However, pydicom always works with Pixel data, it is just that if numpy is installed the data can be converted to a numpy array. I felt like naming it 'pixel' gave the wrong impression. I also thought of 'pixel-arrays' but that seemed long.